Airports/EGSS/SMR/Labels.txt

```diff
@@ -61,8 +61,8 @@
 "42" N051.53.30.500 E000.15.28.630 standHold
 "43" N051.53.32.170 E000.15.25.720 standHold
 "44" N051.53.33.840 E000.15.22.810 standHold
-"45L" N051.53.32.103 E000.15.15.531 standHold
-"45R" N051.53.33.688 E000.15.17.961 standHold
+"45L" N051.53.33.78 E000.15.18.23 standHold
```

Comment:
Notice how these coordinates are shorter than the rest - consistency is key!

And why did you use new coordinates? Surely the only thing that was necessary to change was which label when with which coordinate? :)

Reply:
Hiya Harry,

Yeah I should of just changed the stand numbers around, which I will do when I get home.